apps/prairielearn/src/components/TagDescription.tsx

```diff
@@ -1,15 +1,13 @@
-import { html } from '@prairielearn/html';
-
 import { type Tag } from '../lib/db-types.js';
 
-export function TagDescription(tag: Tag) {
+export function TagDescription({ tag }: { tag: Tag }) {
   if (!tag.implicit) {
-    return tag.description;
+    return <>{tag.description}</>;
```

Comment:
It should be valid to just return the description directly:


```suggestion
    return tag.description;
```